tests/spec/core/utils-spec.js

```diff
@@ -307,22 +307,26 @@ describe("Core - Utils", () => {
     const localizationStrings = {
       en: { foo: "EN Foo", bar: "EN Bar" },
       ko: { foo: "KO Foo" },
+      de: { foo: "DE Foo" },
```

Comment:
Changes in this file are not needed. The changes in test titles are welcome though.

Reply:
This additional data will make "falls back to English string if language does not exit in language data" test fail though.

Reply:
I reverted commit in https://github.com/w3c/respec/pull/2687/commits/980ccd9733fa54161b538b048ed9c637f8c59e39